dev/Gems/EMotionFX/Code/EMotionFX/Source/Importer/Importer.cpp

```diff
@@ -634,6 +634,7 @@ namespace EMotionFX
         // get rid of shared data
         ResetSharedData(sharedData);
         sharedData.Clear();
+        sharedData.Shrink();
```

Comment:
Is the clear/shrink needed since sharedData is going out of scope and will be destroyed?